Formula/s/sq.rb

```diff
@@ -0,0 +1,43 @@
+class Sq < Formula
+  desc "Data wrangler with jq-like query language"
+  homepage "https://sq.io"
+  url "https://github.com/neilotoole/sq/archive/refs/tags/v0.48.3.tar.gz"
+  sha256 "46e75e2db83a6cbc98b07dbcfb23de03fc41b2b2cbc7de7aaee0425cef4fb9bb"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  uses_from_macos "sqlite" => :test
+
+  def install
+    icu4c = Formula["icu4c"]
+    ENV.append "CFLAGS", "-I#{icu4c.opt_include}"
+    ENV.append "LDFLAGS", "-L#{icu4c.opt_lib}"
```

Comment:
Kinda weird to have this when `icu4c` isn't even a dependency?

Reply:
Sorry, forgot to remove those lines.